examples/README.md

```diff
@@ -1,6 +1,6 @@
 # Examples
 
-- [Source to Knative service](./source-to-knative-service): demonstrates how
+- [Source to Knative service](./source-to-gitops): demonstrates how
```

Comment:
```suggestion
- [Source to Gitops](./source-to-gitops): demonstrates how
```
Should we also mention gitops-to-app here?

Reply:
👍